command/runc.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package command
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"slices"
+	"syscall"
+)
+
+type RuncCommand struct {
+}
+
+func (c *RuncCommand) Help() string {
+	return ""
+}
+
+func (c *RuncCommand) Name() string { return "runc" }
+
+func (c *RuncCommand) Run(args []string) int {
+	if slices.Contains(args, "create") && slices.Contains(args, "--bundle") {
+		bundle := args[slices.Index(args, "--bundle")+1]
+
+		jsonFile, _ := os.Open(fmt.Sprintf("%s/config.json", bundle))
+		byteValue, _ := io.ReadAll(jsonFile)
+		jsonFile.Close()
+
+		var config map[string]interface{}
```

Comment:
Not sure, I was hoping for https://github.com/opencontainers/runc but didn't find it immediately in there. I would imagine that `runc` somewhere has it because they most likely not parse it by hand either (might be in a dependency instead though)